Distiller-server/src/main/java/com/mapr/distiller/server/recordtypes/ThreadResourceRecord.java

```diff
@@ -84,78 +141,43 @@ public static boolean produceRecord(RecordQueue thread_resources, String produce
 	public String toString(){
```

Comment:
Here, we can have @Override for toString(), so that whenever we try to print/log it will automatically call the toString(). We don't need to call explicitly.